src/reanimated/useReanimatedHeaderHeight.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import ReanimatedHeaderHeightContext from './ReanimatedHeaderHeightContext';
 
-export default function useReanimatedTransitionProgress() {
+export default function useReanimatedHeaderHeight() {
```

Comment:
I'm not sure right now, but if it was part of public API we should not rename it like that. If it was not, it is ok then.

Reply:
It wasn't yet, as useReanimatedHeaderHeight wasn't in any release and it was my mistake that I didn't renamed the export and function in this hook 😅

Reply:
I see, the change is fine then